packages/phone-number-privacy/combiner/src/match-making/get-contact-matches.ts

```diff
@@ -68,10 +95,10 @@ export async function handleGetContactMatches(
 
 function isValidGetContactMatchesInput(requestBody: GetContactMatchesRequest): boolean {
   return (
+    hasValidBlindedPhoneNumberParam(requestBody) &&
```

Comment:
We will probably have to add this only after Valora versions are updated to pass in this value (will take a while for all old versions to trickle out of use)